apps/viewer/src/app.tsx

```diff
@@ -135,8 +135,8 @@ const App = () => {
             <Text test-id="pageDesc">Please wait for livestream to begin.</Text>
           </VStack>
         ) : (
-          <HStack height="573px" maxHeight="573px">
-            <Box height="100%" test-id="millicastVideo">
+          <HStack height="573px" justifyContent="center" maxHeight="573px" width="100%">
+            <Box height="100%" maxWidth="1024px" test-id="millicastVideo" width="100%">
```

Comment:
Should we use VW instead of fixed pixels here?

Reply:
We don't necessarily want relative sizing here. On larger/smaller screen sizes, relative sizes tend to make things look too big or too small. This would be exacerbated in our case since we may have videos with different resolutions/aspect ratios: if the main stream has a "square" aspect ratio, it would make the side videos too small.

Reply:
Relative size should be better to adapt larger or smaller screen sizes, otherwise have you tested in different screen sizes? Such as 4K or 720p?